src/shared-components/selectorButton/test.tsx

```diff
@@ -82,11 +74,11 @@ describe('<SelectorButton />', () => {
           </SelectorButton>,
         );
 
-        expect(tree).toMatchSnapshot();
+        expect(container.querySelectorAll('svg').length).toBe(0);
```

Comment:
This test, I think, suffers from the problem of unhelpful snapshots. The `it` block is focusing on a very specific part of the test, but it is serializing everything as a snapshot. That's not super clear. This just takes advantage of `container` being a DOM node to use DOM API to check if there's an SVG present in the markup.

Reply:
Currently, we strip the `<title>` from our `SVGs`. If we were to change our [`svgr` settings](https://react-svgr.com/docs/options/#title) to keep the title, we could use the built-in `getByTitle` helper method and check for the presence of the icon name we are passing in.